lib/linguist/heuristics.yml

```diff
@@ -365,7 +365,7 @@ disambiguations:
   - language: Win32 Message File
     pattern: '(?i)^[ \t]*(?>\/\*\s*)?MessageId=|^;'
   - language: M4
-    pattern: '^dnl|^divert\((?:-?\d+)?\)|^\w+\(`[^\n]*?'[),]'
+    pattern: '^dnl|^divert\((?:-?\d+)?\)|^\w+\(`[^\n]*?''[),]'
```

Comment:
I swear I can never remember what YAML's frikkin' syntax is for escaping single-quotes… 🤦‍♂️